<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="53" total="55" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23858" started-at="2019-02-12T15:02:08Z" finished-at="2019-02-12T15:02:32Z">
    <groups>
      <group name="BeFan">
        <method signature="BeFANMayorista.TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f]" name="TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil" class="Tests.BeFANMayorista"/>
        <method signature="BeFANMayorista.TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f]" name="TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR" class="Tests.BeFANMayorista"/>
      </group> <!-- BeFan -->
    </groups>
    <test name="Default test" duration-ms="23858" started-at="2019-02-12T15:02:08Z" finished-at="2019-02-12T15:02:32Z">
      <class name="Tests.BeFANMayorista">
        <test-method status="PASS" signature="init()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f]" name="init" is-config="true" duration-ms="23251" started-at="2019-02-12T15:02:08Z" finished-at="2019-02-12T15:02:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f]" name="TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil" duration-ms="200" started-at="2019-02-12T15:02:31Z" finished-at="2019-02-12T15:02:31Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[C:\BefanArchivos\salida\Resultado201902120.txt (The system cannot find the file specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: C:\BefanArchivos\salida\Resultado201902120.txt (The system cannot find the file specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileReader.<init>(Unknown Source)
at Pages.BeFan.soyEzpesial(BeFan.java:218)
at Tests.BeFANMayorista.TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil(BeFANMayorista.java:1681)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil -->
        <test-method status="SKIP" signature="TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f]" name="TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR" duration-ms="0" started-at="2019-02-12T15:02:32Z" depends-on-methods="Tests.BeFANMayorista.TS97654_BeFan_Movil_REPRO_PreaActivacion_Linea_Repro_Localidad_inexistente_para_numeracion_movil" finished-at="2019-02-12T15:02:32Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method BeFANMayorista.TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method BeFANMayorista.TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR()[pri:0, instance:Tests.BeFANMayorista@74ad1f1f] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TS97670_BeFan_Movil_REPRO_Seriales_con_estado_ERROR -->
      </class> <!-- Tests.BeFANMayorista -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
